### PR TITLE
[stable/kubernetes-dashboard]: Remove default pod resource settings

### PR DESCRIPTION
**Pull request by bnutt**
#### What this PR does / why we need it:

Removes the default pod resource limits and requests.

Make it so users have the option to set resource configuration for their pods.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/kubernetes-dashboard/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: kubernetes-dashboard
-version: 1.8.0
+version: 1.8.1
```
Comment (kfox1111) [inline stable/kubernetes-dashboard/Chart.yaml:3]: I'm ok with this, but it is a breaking change. There's a pr in the works for 2.0.0 that will be a major breaking change. Maybe we should apply this there?

Comment (bnutt) [reply stable/kubernetes-dashboard/Chart.yaml:3]: Do you suggest to merge those changes in with the pr so it can be part of the 2.0.0 release?

**Comment by desaintmartin** [comment]
I am curious, why do you want to remove those defaults? Do you consider it doesn't work for most people?

**Pull request closed without merging**